Tella/Data/Database/Common/DatabaseConstants.swift

```diff
@@ -13,7 +13,7 @@ struct D {
     
     /* DATABASE VERSION */
 
-    static let databaseVersion = 5
+    static let databaseVersion = 7
```

Comment:
We don't need to upgrade the `databaseVersion` because the version has not published yet on prod, so we need to keep it to `6`